### PR TITLE
mob pack

### PR DESCRIPTION
**Pull request by Tenscore**

### CODE REVIEW

**Review by SewerCzar — APPROVED** [review]

File: Content/Data/Body-External.xml [hunk 1]
```diff
@@ -130,11 +130,314 @@
       <frame>(0,0,70,70)</frame>
       <frame>(70,0,70,70)</frame>
       <frame>(140,0,70,70)</frame>
+<<<<<<< HEAD
+    </frames>
+    -->
+=======
     </frames> -->
+>>>>>>> 387067d322230cee3b18cf96ce8b229f6c5435a3
```
Comment (Chilly021) [inline Content/Data/Body-External.xml:138]: Same

File: Content/Data/Body-External.xml [hunk 2]
```diff
@@ -130,11 +130,314 @@
...
   </body>
   <!-- Red Drake -->
   <body id="320" corpse="5320">
     <texture>Azimuth\monsters\drakes\drake_red</texture>
     <source>(0,0,90,90)</source>
   </body>
+<<<<<<< HEAD
```
Comment (Chilly021) [inline Content/Data/Body-External.xml:145]: Same

File: Content/Data/Body-External.xml [hunk 3]
```diff
@@ -130,11 +130,314 @@
...
+  <body id="378" corpse="5377">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(60,60,60,60)</source>
+  </body>
+  <!--Orc Praetorian Raider w Kopesh and Shield-->
+  <body id="379" corpse="5377">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(120,60,60,60)</source>
+  </body>
+=======
```
Comment (Chilly021) [inline Content/Data/Body-External.xml:441]: Merge line endings are here. Need to clean those up and make sure the xml is valid.

File: Content/Data/Body-External.xml [hunk 4]
```diff
@@ -130,11 +130,314 @@
...
+>>>>>>> 387067d322230cee3b18cf96ce8b229f6c5435a3
```
Comment (Chilly021) [inline Content/Data/Body-External.xml:442]: Same

File: Content/Data/Items-External.xml [hunk 5]
```diff
@@ -529,4 +529,43 @@
     <source>(0,0,45,45)</source>
     <offset>(-10,-10)</offset>
   </item>
+<<<<<<< HEAD
```
Comment (Chilly021) [inline Content/Data/Items-External.xml:532]: Merge line endings are here. Need to clean those up and make sure the xml is valid.

File: Content/Data/Items-External.xml [hunk 6]
```diff
@@ -529,4 +529,43 @@
...
+    <texture>Tenser/monsters/praetorian_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+  <!--Orc Praetor Body (trimmed)-->
+  <item id="5377" category="16">
+    <texture>Tenser/monsters/praetorian2_dead</texture>
+    <source>(0,0,45,45)</source>
+  </item>
+</data>
+=======
```
Comment (Chilly021) [inline Content/Data/Items-External.xml:569]: Merge line endings are here. Need to clean those up and make sure the xml is valid.

File: Content/Data/Items-External.xml [hunk 7]
```diff
@@ -529,4 +529,43 @@
...
 </data>
+>>>>>>> 387067d322230cee3b18cf96ce8b229f6c5435a3
```
Comment (Chilly021) [inline Content/Data/Items-External.xml:571]: Merge line endings are here. Need to clean those up and make sure the xml is valid.

**Review by Chilly021 — CHANGES_REQUESTED** [review]

**Review by Chilly021 — APPROVED** [review]

**Review by zrikzlok — APPROVED** [review]

**Pull request merged**